specification/hybridnetwork/resource-manager/Microsoft.HybridNetwork/stable/2023-09-01/examples/NetworkFunctionDefinitionVersionCreate.json

```diff
@@ -48,12 +48,12 @@
                   "options": {
```

Comment:
>options [](http://example.com/codeflow?start=19&length=7)

now I look at docs for this [https://helm.sh/docs/helm/helm_upgrade/](https://helm.sh/docs/helm/helm_upgrade/)

I am kinda surprised the API isn't modeled more carefully as a conventional REST API, with enums/types/units in the names etc, instead of these string-ified-booleans.

e.g. it could have been 

```
                    "installOptions": {
                      "atomic": "Atomic" | "Nonatomic",
                      "wait": "ForTimeout" | "ForJobs",
                      "timeout": "PT30M"
```

But you can't change that except as a breaking change in a future api version.